package.json

```diff
@@ -43,11 +53,11 @@
     "bs58check": "^2.1.2",
     "buffer": "^5.6.0",
     "crc-32": "^1.2.0",
-    "dotenv": "^16.0.0",
     "elliptic": "6.5.4",
     "eth-eip712-util-browser": "^0.0.3",
     "hash.js": "^1.1.7",
     "js-sha3": "^0.8.0",
+    "patch-package": "^6.4.7",
```

Comment:
What's the reason for this?

Reply:
Ah okay. Seems like it should be in dev deps lol

Reply:
It does get called in `postinstall` but I think that can still use devdeps right?

Reply:
Sorry--missed this. It's been moved [851bf2e](https://github.com/GridPlus/gridplus-sdk/pull/439/commits/851bf2e208c855edd204037c696f211d1510146e)